test/generators/component_generator_test.rb

```diff
@@ -13,4 +13,12 @@ class ComponentGeneratorTest < Rails::Generators::TestCase
     assert_file "app/components/foobar/foobar.css"
     assert_file "app/components/foobar/foobar.js"
   end
+
+  test "component generator with sub component" do
```

Comment:
Perhaps `component generator for namespaced component`?

Reply:
Good call, I meant to change that to something more sensible, but forgot.